MekHQ/src/mekhq/resources/CampaignOptionsDialog.properties

```diff
@@ -34,6 +34,8 @@ resetToFirstTechCheckBox.text=After repair reset to the technician at the top of
 resetToFirstTechCheckBox.toolTipText=After any repair/salvage task, always selects the technician at the top of the list\n(Can be combined with option to place tech assigned to unit at top of list)
 useDamageMargin.text=Damage/destroy parts by margin of failure
 useDamageMargin.toolTipText=Instead of destroying/damaging parts only by elite techs, parts are damaged/destroyed when the tech roll\nfails by a given margin.
+useAeroSystemHits.text=(Unofficial) Damage/destroy Aero system parts by number of hits taken
```

Comment:
Maybe replace (Unofficial) with (Proposed Errata) to make it clear to people what is going on here